.travis.yml

```diff
@@ -5,17 +5,34 @@ go:
   - 1.6
   - 1.7rc1
 
+cache:
+  directories:
+    - admin/public/node_modules
+
 services:
   - redis-server
 
-before_script:
-  - go vet $(go list ./... | grep -v /vendor/)
+env:
+  global:
+    - GO15VENDOREXPERIMENT=1
+  matrix:
+    - TEST=UI
+    - TEST=DEFAULT
+
+matrix:
+  exclude:
+    - go: 1.5
+      env: TEST=UI
+    - go: 1.7rc1
```

Comment:
Just wanted to run them once since the go version doesn't matter, I'll see if there's a better way to set up the matrix.